make_site.py

```diff
@@ -364,9 +373,9 @@ def download_N_theorems(kind: NTheorems) -> dict:
                             # note: the `header-data.json` data file uses doc-relative links
                             docs_link='/mathlib4_docs/' + decl_info.info.docLink,
                             src_link=decl_info.info.sourceLink))
-                # A theorem counts as formalized if the author field or `doc_decls` is non-empty.
-                formalized = bool(h.author) or (len(doc_decls) > 0)
-                theorems.append(TheoremForWebpage(id, h.title, formalized, doc_decls, links, h.author, h.date, note))
+                # A theorem's proof's proof counts as formalized if the author field or `doc_decls` is non-empty.
+                proof_formalized = bool(h.author) or (len(doc_decls) > 0)
+                theorems.append(TheoremForWebpage(id, h.title, statement_formalised, proof_formalized, status, doc_decls, links, h.author, h.date, note))
```

Comment:
I'm a little confused by these lines. First, is there an extra "proof" in the comment? Second, if `h.statement` was present, won't it get added to `doc_decls`, thus making this `proof_formalized` also `True`? Finally, I'm not sure what `status` is meant to be but it's undefined.

Reply:
Good catch! `proof_formalized` was a logic error; fixed now by re-ordering the logic.
The extra "proof" and `status` are merge errors, it seems: corrected.